Documentation/README.md

```diff
@@ -220,6 +221,10 @@ Viewpoints are immutable, therefore they should never be changed once created. I
 ## Visualization information (.bcfv) file
 The visualization information file contains information of components related to the topic, camera settings, and possible markup and clipping information.
 
+### TopicFiles
+
+Parent topic objects specify a list of model files in their `Header.Files` element. These files contain an `Id` attribute, which can be used to optionally reference a subset of all the topic files for a specific viewpoint. If the viewpoint does contain entries in it's `TopicFiles` element, then only those files referenced there should be loaded when visualizing the viewpoint. Otherwise, all files from the topic should be loaded.
+
```

Comment:
We should change it so that viewpoints need to list their models, otherwise adding models to the topic would result in the viewpoints changing.

Reply:
Resolved via fd8c275b048291112793247e76bd9a50a6fc65e9